.github/workflows/ci.yaml

```diff
@@ -16,6 +16,14 @@ jobs:
       with:
         scandir: './bin'
 
+  markdown-lint:
```

Comment:
this job should be a dependency of the `tests` job down below?

Reply:
would be good, yes, will update